test/extended/cluster/cl.go

```diff
@@ -241,16 +246,34 @@ func newProject(nsName string) *projectapi.Project {
 	}
 }
 
-// mkPath returns fully qualfied file location as a string
-func mkPath(file string) string {
+// mkPath returns fully qualfied file path as a string
+func mkPath(filename, config string) (string, error) {
+	// Use absolute path if provided in config
+	if filepath.IsAbs(filename) {
+		return filename, nil
+	}
 	// Handle an empty filename.
-	if file == "" {
-		e2e.Failf("No template file defined!")
+	if filename == "" {
+		return "", fmt.Errorf("No template file defined!\n")
+	}
+
+	var searchPaths []string
+	workingDir, err := os.Getwd()
+	if err != nil {
+		return "", err
 	}
-	if rootDir == "" {
-		rootDir = "content"
+	configDir := filepath.Dir(config)
+
+	searchPaths = append(searchPaths, filepath.Join(workingDir, filename))
+	searchPaths = append(searchPaths, filepath.Join(configDir, filename))
```

Comment:
Building a slice of the two search paths and then searching both rather than just searching each as they're created.